<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>950</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: transparent;</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>800</width>
      <height>620</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>620</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* Apply this ONLY to the Frame sitting on top */
QFrame {
    background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, 
        stop:0.45 rgba(255,255, 255, 255),  
        stop:0.8 rgba(0, 0, 0, 0), 
        stop:1 rgba(0, 0, 0, 0)
    );
    border-radius: 10px;
	border: none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>531</width>
        <height>313</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter Tight</family>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Inter Tight&quot;;

	background-color: transparent;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLineEdit" name="utilisateur_lineedit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>480</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: transparent;
    border: 2px solid #4C5363; /* Soft gray instead of transparent for better visibility */
    border-radius: 10px;
    color: #0d0c22;
    font-size: 16px;
    height: 35px;
    /* Extra left padding to make room for an icon */
    padding-left: 10px; 
    padding-right: 0px;
}

QLineEdit:hover {
    border-color: #818cf8;
    background-color: #ffffff;
}

QLineEdit:focus {
    border-color: #818cf8;
    background-color: #ffffff;
    /* Qt doesn't support spread/blur shadows, so we use a solid highlight */
    border: 2px solid #818cf8;
}

/* Placeholder color is handled specially in QSS */
QLineEdit {
    qproperty-placeholderText: &quot;Utilisateur&quot;;
}
font: 57 13pt &quot;Inter Tight&quot;;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 16pt &quot;Inter Tight&quot;;</string>
       </property>
       <property name="text">
        <string>Utilisateur</string>
       </property>
      </widget>
      <widget class="QLabel" name="connection_error_label">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>195</y>
         <width>300</width>
         <height>18</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;MS Gothic&quot;;
font: 8pt &quot;ae_Ostorah&quot;;
color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>utilisateur ou mot de pass est incorrect!</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mot_de_pass_lineedit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>480</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: transparent;
    border: 2px solid #4C5363; /* Soft gray instead of transparent for better visibility */
    border-radius: 10px;
    color: #0d0c22;
    font-size: 16px;
    height: 35px;
    /* Extra left padding to make room for an icon */
    padding-left: 10px; 
    padding-right: 0px;
}

QLineEdit:hover {
    border-color: #818cf8;
    background-color: #ffffff;
}

QLineEdit:focus {
    border-color: #818cf8;
    background-color: #ffffff;
    /* Qt doesn't support spread/blur shadows, so we use a solid highlight */
    border: 2px solid #818cf8;
}

/* Placeholder color is handled specially in QSS */
QLineEdit {
    qproperty-placeholderText: &quot;Mot de pass&quot;;
}
font: 57 13pt &quot;Inter Tight&quot;;
</string>
       </property>
       <property name="inputMask">
        <string comment="*"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Mot de pass</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>350</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 16pt &quot;Inter Tight&quot;;</string>
       </property>
       <property name="text">
        <string>Mot do pass</string>
       </property>
      </widget>
      <widget class="QPushButton" name="se_connecter_button">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>250</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #1D7FD8;
    border: 0px solid #000000;
    border-radius: 10px;
    color: #ffffff;
    font-weight: bold;
    font-size: 13px;
    /* letter-spacing is simulated with spaces in the button text */
    padding: 10px 0px;
    text-transform: uppercase;
}

QPushButton:hover {
    /* We create a hard-stop gradient that looks like a solid fill */
    background-color: #1769B4;
    color: #FFFFFF;
    border: 2px solid #1D7FD8;
}

QPushButton:pressed {
    background-color: #0E5891;
	color: #ffffff;
    border-color: #1D7FD8;
}
font: 57 14pt &quot;Inter Tight&quot;;</string>
       </property>
       <property name="text">
        <string>Se Conecter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="annuler_button">
       <property name="geometry">
        <rect>
         <x>193</x>
         <y>250</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: transparent;
    border: 0px solid #000000;
    border-radius: 10px;
    color: #1D7FD8;
    font-weight: bold;
    font-size: 13px;
    /* letter-spacing is simulated with spaces in the button text */
    padding: 10px 0px;
    text-transform: uppercase;
}

QPushButton:hover {
    /* We create a hard-stop gradient that looks like a solid fill */
    background-color: #1769B4;
    color: #ffffff;
    border: 2px solid #1D7FD8;
}

QPushButton:pressed {
    background-color: #0E5891;
	color: #ffffff;
    border-color: #1D7FD8;
}
font: 57 16pt &quot;Inter Tight&quot;;</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(&quot;program/assets/global/erp_logo.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>471</width>
       <height>531</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-bottom-left-radius: 50px</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>951</width>
       <height>620</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	border-image: url(&quot;program/assets/login/login_img.jpg&quot;);
	/*border-image: url(:/assets/program/assets/login/login_img.jpg);*/
    /* Ensures the image fills the widget entirely */
    background-position: center;
    background-repeat: no-repeat;
    
    /* This is the secret to getting rid of those side-bars */
    background-origin: content;
    background-clip: border;
    
    border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_4</zorder>
    <zorder>label_3</zorder>
    <zorder>frame</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
